components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -129,9 +130,9 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
          */
 
         String redirectURL;
-        Cookie opBrowserStateCookie = OIDCSessionManagementUtil.getOPBrowserStateCookie(request);
+        String opBrowserStateCookieValue = getOPBSCookieValue(request);
 
-        if (opBrowserStateCookie == null) {
+        if (opBrowserStateCookieValue == null) {
```

Comment:
Can Use StringUtils? WDYT?

Reply:
Yes. We can. It's fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1519/commits/b6b16b4e9375169a8e8109dead588f7651e3a54b.